.github/TESTING.md

```diff
@@ -0,0 +1,22 @@
+## Testing Expectations
+All Pull Requests are expected to include sufficient testing.
+
+- If a new component has been added, a unit test for that component should be include
```

Comment:
changed it to 'a unit test for that component should be created' in order to avoid repeating the word included because it seemed odd when I read it back